README.md

```diff
@@ -1 +1,62 @@
-# Todo_review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list
+
+> This project is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## Demo 
+Click here to see [Live](https://lionrouge1.github.io/To_do_list/)
+
+## Built With
+
+- HTML
+- Javascript
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+- Open Terminal.
+- Change the current working directory to the location you want the cloned directory.
+- Enter the following:
+
+```
+git@github.com:LionRouge1/To_do_list.git
+
+```
+
+- Press Enter to create your local clone.
+- run ``cd To_do_list``
+- now you an open the project with any code editor and go live or simply open the `dist/index.html` page with the browser.
+
+### Prerequisites
+
+- GitHub
+- Git
+
+## Authors
```

Comment:
- [ ] please change this from authors to author

Reply:
👍

Reply:
I have noticed you forgot to change this one from Authors to Author